LLama.KernelMemory/LLamaSharp.KernelMemory.csproj

```diff
@@ -1,9 +1,30 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Are we intentionally dropping/not including netstandard2.0 support for kernel memory?

I'm not against that, just wanted to make sure it's deliberate!

Reply:
This a line that was removed! 🤣 What I was doing is adding the frameworks including netstandard2.0. I think we won't drop netstandard2.0 because it has many users.

Reply:
Update: Though we want to support netstandard2.0, kernel-memory does not... I removed it and will add it back in the future if kernel-memory developers change the mind.